vendor/assets/javascripts/jquery-querytojson/querytojson.js

```diff
@@ -0,0 +1,13 @@
+jQuery.extend({
+  getQueryParameters : function(str) {
```

Comment:
where is this being used?

Reply:
I don't know what this is, it might not be used